tkltest/generate/ui/templates/GeneratedTests.jinja

```diff
@@ -62,6 +62,11 @@ public class GeneratedTests {
 
         // create wait instance
         this.webDriverWait = new WebDriverWait(webDriver, Duration.ofMillis(waitAfterEvent));
+
+{% if precrawl_actions %}
```

Comment:
@sinha108 why is it enough to perform pre-crawl actions before every test class, when before every test method we reload the base URL?

Reply:
@rachelt44 The pre-crawl actions are meant to be executed when the browser is first launched via web driver. This feature does not handle actions that need to run each time the AUT URL is opened; we have not seen the need for that so far, but support for actions to be performed on each AUT URL load can be added.